sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample13_AdvancedConfiguration.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample13_AdvancedConfiguration : ServiceBusLiveTestBase
+    {
+        public void ConfigureProxy()
```

Comment:
These be tagged with `[Test]` so that they're executed nightly.

Reply:
These aren't asserting anything. They will still be built as part of CI, but there is no need to execute them.

Reply:
My understanding is that the goal of snippets was to try to make sure that the code compiles and can be run - acting as smoke tests.   I don't feel particularly strongly in this case, but it seems to drift from the intention.

Reply:
I think compiles is the main thing. The run part necessitates conditionals to set the connection string appropriately. IMO it is not worth the added friction for small snippets like this. I also don't feel too strongly either way, but I'd prefer to err on the side of making it easier to create samples.

Reply:
I'd also add that we have coverage for all of these scenarios in unit and live tests.

Reply:
Ended up switching over to tests.